include/api.php

```diff
@@ -38,14 +37,14 @@ function pll_the_languages( $args = array() ) {
 }
 
 /**
- * Returns the current language on frontend
- * Returns the language set in admin language filter on backend ( false if set to all languages )
+ * Returns the current language on frontend.
+ * Returns the language set in admin language filter on backend ( false if set to all languages ).
  *
  * @api
  * @since 0.8.1
  *
- * @param string $field Optional, the language field to return ( see PLL_Language ), defaults to 'slug', pass OBJECT constant to get the language object.
- * @return string|PLL_Language|bool The requested field for the current language
+ * @param string $field Optional, the language field to return ( @see PLL_Language ), defaults to 'slug'. Pass OBJECT constant to get the language object.
```

Comment:
We should choose between using two of our own constants, or two strings.

Reply:
Sorry, I don't understand.